pkg/scheduler/util/utils.go

```diff
@@ -56,7 +56,7 @@ func GetPodStartTime(pod *v1.Pod) *metav1.Time {
 func GetEarliestPodStartTime(victims *extenderv1.Victims) *metav1.Time {
 	if len(victims.Pods) == 0 {
 		// should not reach here.
-		klog.Errorf("victims.Pods is empty. Should not reach here.")
+		klog.ErrorS(nil, "Victims.Pods is empty. Should not reach here.")
```

Comment:
does this need to be changed? it's referring to a variable name here

Reply:
Any better suggestion?
According to structured Logging migration,Errorf needs to be converted to ErrorS.

Reply:
nvm, I'm fine with this